### PR TITLE
Migrate some elections RPC hooks to SDK 

### PR DESCRIPTION
**Pull request by selankon**
- [x] Migrate elections list page
- [x] Migrate elections count

Tasks before merge:

- [x] https://github.com/vocdoni/vocdoni-sdk/pull/174
- [x] New SDK version published
- [x] ~~Merge~~ Rebase  https://github.com/vocdoni/explorer-ui/pull/130/  and rebase master in order to delete the `useProcessCount` hook
- [x] Todo on `components/blocks/card/process-card.tsx`
- [x] Check it works after rebased from #130

### CODE REVIEW

File: components/blocks/card/entity-card.tsx [hunk 1]
```diff
@@ -12,25 +12,25 @@ import { ReducedTextAndCopy } from '@components/blocks/copy-button';
 import { BodyWrapper, CardItemSubTitle, CardItemTitle, GenericCardWrapper } from '@components/elements/card-generic';
 import { BreakWord } from '@components/elements/styled-divs';
 import { useOrganization } from '@vocdoni/chakra-components';
+import { ensure0x } from '@vocdoni/common';
```
Comment (elboletaire) [inline components/blocks/card/entity-card.tsx:15]: Import it from the SDK instead, this way we can stop using old packages.

File: i18n/locales/ca.json [hunk 2]
```diff
@@ -261,6 +261,7 @@
     "filter": {
       "search": "Cercar",
       "show_only_processes_with_results": "Mostrar només processos amb resultats",
+      "search_by_processId_or_organization": "Cerca per procés o organització",
```
Comment (elboletaire) [inline i18n/locales/ca.json:264]: That `underscore_withCamel` is kinda weird.

**Review by elboletaire — CHANGES_REQUESTED** [review]

File: components/blocks/card/entity-card.tsx (hunk 1 above)
Comment (selankon) [reply components/blocks/card/entity-card.tsx:15]: e74c5b6

File: i18n/locales/ca.json (hunk 2 above)
Comment (selankon) [reply i18n/locales/ca.json]: 339822b

**Pull request merged**